### PR TITLE
[6.12.z] Remove redundant rhev provisioning test

### PR DESCRIPTION
**Pull request by Gauravtalreja1**
Cherrypick of https://github.com/SatelliteQE/robottelo/pull/11737
Fixes https://github.com/SatelliteQE/robottelo/issues/11774

### CODE REVIEW

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/api --uses-fixtures setting_update

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/api --uses-fixtures setting_update

File: tests/foreman/longrun/test_provisioning_computeresource.py [hunk 1]
```diff
@@ -89,89 +88,6 @@ def tear_down(provisioning):
         Host.delete({'id': host['id']})
 
 
-@pytest.mark.on_premises_provisioning
-@pytest.mark.vlan_networking
-@pytest.mark.tier3
-def test_positive_provision_rhev_with_host_group(rhev, provisioning, target_sat, tear_down):
```
Comment (jyejare) [inline tests/foreman/longrun/test_provisioning_computeresource.py:95]: Why does this test is removed, is it expected ?

File: tests/foreman/ui/test_host.py [hunk 2]
```diff
@@ -251,16 +251,22 @@ def module_activation_key(module_org_with_manifest, module_target_sat):
...
-@pytest.fixture(scope='function')
-def remove_vm_on_delete(target_sat, setting_update):
-    setting_update.value = 'true'
-    setting_update.update({'value'})
-    assert (
-        target_sat.api.Setting().search(query={'search': 'name=destroy_vm_on_host_delete'})[0].value
-    )
-    yield
+@pytest.fixture
+def tracer_install_host(rex_contenthost, target_sat):
```
Comment (jyejare) [inline tests/foreman/ui/test_host.py:255]: I see the def is replacing another def but doesnt seems we are chaning the usage of this replaced function.

Expected ?

**Review by jyejare — CHANGES_REQUESTED** [review]

File: tests/foreman/longrun/test_provisioning_computeresource.py (hunk 1 above)
Comment (Gauravtalreja1) [reply tests/foreman/longrun/test_provisioning_computeresource.py:95]: Yes, it is already covered https://github.com/SatelliteQE/robottelo/blob/6.12.z/tests/foreman/cli/test_computeresource_rhev.py#L383

File: tests/foreman/ui/test_host.py (hunk 2 above)
Comment (Gauravtalreja1) [reply tests/foreman/ui/test_host.py:255]: Oh, I missed this while resolving conflicts, removed it, Thanks!

**Review by Griffin-Sullivan — APPROVED** [review]

**Review by shubhamsg199 — APPROVED** [review]

**Review by synkd — APPROVED** [review]

**Review by JacobCallahan — APPROVED** [review]

**Pull request merged**